test/production/standalone-mode/required-server-files/required-server-files.test.ts

```diff
@@ -24,14 +25,9 @@ describe('required server files', () => {
   let errors = []
   let stderr = ''
   let requiredFilesManifest
+  let minimalMode = true
```

Comment:
Can we pass this to `setupNext` instead? It's already hard enough to follow these these tests. Let's not make it worse with more test abstractions. WET tests are fine.

Reply:
I don't think it's that straightforward, because the next instance is created once for all tests, and the minimalMode is supposed to be changed for only one of them.

Reply:
Apart from `patchServerFile` I didn't really add more abstractions, I basically just moved the `initNextServerScript` call from `beforeAll` to `beforeEach`. I can also inline `patchServerFile`, and revert the usage in the three other test files, if you prefer.

Reply:
> I can also inline patchServerFile, and revert the usage in the three other test files, if you prefer.

@eps1lon I did do this, by the way. Any other concerns with the PR as it is right now?